### PR TITLE
add the initial scripts.

### PR DESCRIPTION
**Pull request by maximilianpress**
I refactored the following to some extent:
* makeAgpFromFasta.py (from Shawn's)
* agp2assembly.py (from Perl)

But not the following:
* juicebox_assembly*.py
* contigFunctions.py

### CODE REVIEW

**Comment by bnelsj** [comment]
I added a note that only Python 2 is supported.

Duplicating my other comments here:
I think this is useful for now, but there's a lot we could do to improve the repo. CI, use of common libraries (argparse, Biopython, etc.), and python 3 support would be my top 3.

File: src/juicebox_assembly_converter.py [hunk 1]
```diff
@@ -0,0 +1,385 @@
+#!usr/bin/python
```
Comment (bnelsj) [inline src/juicebox_assembly_converter.py]: This is more portable and explicit:
`#!/usr/bin/env python2`
See https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take

File: test/test_juicebox_assembly_converter.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+class JuiceboxConverterTestCase(unittest.TestCase):
+    def setUp(self):
+        self.converter = JuiceboxConverter()
+        self.test_output_fasta = 'test_output.fasta'
+        self.test_output_agp = 'test_output.agp'
+        self.test_output_bed = 'test_output.bed'
+        self.test_output_break_report = 'test_output.break_report.txt'
+        
+        self.collateral_dir = 'test/collateral/'
```
Comment (bnelsj) [inline test/test_juicebox_assembly_converter.py]: Build an absolute path from the __file__ variable like this:
`os.path.dirname(__file__) + '/test/collateral/`

**Comment by shawnpg** [comment]
Revamped the entire stack for generating .fasta, .bed, and .agp from Juicebox .assembly files. Added unit tests. Minor usage updates to the other two AGP scripts. Updated readme

**Review by zeeev — APPROVED** [review]

**Review by shawnpg — APPROVED** [review]

**Review by bnelsj — APPROVED** [review]
Tested and approved.

**Pull request merged**